app/database_etl/etl_main.py

```diff
@@ -348,22 +359,38 @@ def apply_study_max_estimate_grade(df):
     return df
 
 
-# Returns a list of 'city,state' if we
+# Returns a list of 'city,state_countryCode' if we
 # can associate a city with a state, else
 # returns a list of cities
+# This is necessary to properly geosearch cities
+# as there can be multiple cities of the same name in a country
 def get_city(row):
     if row['city']:
         cities = row['city'].split(',')
         # if only 1 state associated with the record
         # associate the city with the state
         # so that we can get a pin for it
         if row['state'] and len(row['state']) == 1:
-            cities = [f"{city},{row['state'][0]}" for city in cities]
+            cities = [f"{city},{row['state'][0]}_{get_country_code(row['country'], iso3=False)}"
+                      if get_country_code(row['country'], iso3=False)
+                      else f"{city},{row['state'][0]}" for city in cities]
+
         return cities
     else:
         return row['city']
 
 
+# Returns "state_countryCode"
+# Needed becuase country code is used to limit mapbox API
+# geosearch queries (improving query accuracy)
+def add_country_code_to_state(row):
+    if row['state']:
+        return [f"{state}_{get_country_code(row['country'], iso3=False)}"
```

Comment:
can you just make this a for loop and return a variable? it's a bit long of a list comp

Reply:
sure thing